trento/xml/article_sap_trento.xml

```diff
@@ -1678,618 +1677,48 @@ In the Trento dashboard, go to the overview corresponding to the object for whic
-        entry in the left sidebar.
-      </para>
-      <figure>
-        <title><guimenu>Checks catalog</guimenu></title>
+      </figure>  
+   <para>Once the SUSE Manager settings are entered the SAP admin can test the 
+   connection by clicking on the Test Connection button. If the connection is successfull,
+   from that moment onward, for any registered host that is also managed by the SUSE Manager
+   instance which connection details we have provided, the corresponding Host Details view will
+   display a summary of the available patches and the upgradable packages for that particular host:</para>
+        <figure>
+         <title><guimenu>Availale software updates in the Host Details view</guimenu></title>
         <mediaobject>
           <imageobject>
-            <imagedata width="80%"
-              fileref="trento-web-settings-catalogs.png" />
+            <imagedata
+              fileref="trento-summary-of-available-software-updates.png" />
           </imageobject>
         </mediaobject>
       </figure>
-      <para>
-        The view allows you to select the platform which values you
-        want to review: Azure, AWS, or GCP. For each platform, it shows
-        the configuration checks that Trento can perform for the different
-        cluster components: &corosync;, &pace;, SBD, Azure Fence Agent,
-        &hana; System Replication Resource Agent, OS and package version,
-        and miscellaneous.
-      </para>
-      <para> Checks are grouped by components. Each test is identified by
-        a test ID, next to which we can see the check description and an
-        info icon. By clicking on the info icon we get detail
-        information about the corresponding check: abstract, remediation
-        actions (what to do when the check is negative) and related
-        references. </para>
-      <para> What checks are carried out for each particular cluster is
-        determined using the <guimenu>Settings</guimenu> button in the
-        corresponding cluster detail view. See <xref
-          linkend="lst-trento-web-pace-cluster-settings" /> for
-        details. </para>
+   <para>There are three types of available patches: security advisories, bug fixes and feature
+   enhancement. Security advisory are regarded as critical and the existance of just one of them
+   will make the health of the host go critical. The others are regarding as warnings.</para>
+    <para>At any given moment, the user can clear the SUSE Manager settings from the Settings view, which
+    will result in all the information regarding available software updates gone from the console and the
+    health of the hosts adjusted accordingly.</para>
+   <para>This integration with SUSE Manager is a first step in road which we expect will take Trento
+   to a point where the SAP admin is able  to orchestrate the patching of the Operating System of their
+   SAP environment using Trento and SUSE Manager behind it.</para>
```

Comment:
This sentence is about the future. I find this very problematic for the following reasons:

* It may be valid now, but who knows what will happen in 3 months or 1 year?
* Roadmaps can change.
* Technical documentation is about facts, not vague promises. :wink:  We document the current state, not any future impact.
* Could be a potential legal threat if a a customer want to hold us reliable for missing features.

As a conclusion, I would really recommend to remove it. It doesn't add any value or could potentially dangerous.

If you really think the user should know about future development, maybe link to the GitHub repo?

Reply:
Let's remove it then.